pipeline.py

```diff
@@ -0,0 +1,72 @@
+import pmu,common
+from tabulate import tabulate
+import itertools,sys,re
+
+def pipeline_view(log,depths):
+  out_pipe=open(log.replace('csv','pipeline.log'),'w');
+  pipe_list=get_log(log)
```

Comment:
Use common here directly, I don't see a need to define a one-line function.

Reply:
fixed